Makefile

```diff
@@ -34,10 +34,13 @@ WARNFLAGS	:= -Wall -pedantic
 
 # Overridable CFLAGS
 CFLAGS		?= -O3 -flto -DNDEBUG
+CXXFLAGS	?= -O3 -flto -DNDEBUG
 # Non-overridable CFLAGS
 # _ISOC11_SOURCE is required on certain platforms to get C11 on top of the C99-based POSIX 2008
 REALCFLAGS	:= ${CFLAGS} ${WARNFLAGS} -std=gnu11 -I include \
 		   -D_POSIX_C_SOURCE=200809L -D_ISOC11_SOURCE
+REALCXXFLAGS	:= ${CXXFLAGS} ${WARNFLAGS} -std=c++17 -I include \
+		   -D_POSIX_C_SOURCE=200809L -fno-exceptions -fno-rtti
```

Comment:
If the C++ tools aren't using POSIX-specific features which depend on `_POSIX_C_SOURCE`, then this can be omitted. I don't think rgbgfx needs anything like that.

Reply:
RGBGFX currently isn't, but at least RGBASM will, so I put it there so it won't be forgotten then. Also, it helps keep the flags symmetrical, imo.